cannot build pixie on Ubuntu 15.10 64bit

Below is the error output. This happens with both `make build_with_jit` and `make build_no_jit`.

```
[translation:info] Error:
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/translator/goal/translate.py", line 318, in main
[translation:info]     drv.proceed(goals)
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/translator/driver.py", line 550, in proceed
[translation:info]     result = self._execute(goals, task_skip = self._maybe_skip())
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/translator/tool/taskengine.py", line 114, in _execute
[translation:info]     res = self._do(goal, taskcallable, *args, **kwds)
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/translator/driver.py", line 279, in _do
[translation:info]     res = func()
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/translator/driver.py", line 346, in task_rtype_lltype
[translation:info]     rtyper.specialize(dont_simplify_again=True)
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 174, in specialize
[translation:info]     self.specialize_more_blocks()
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 214, in specialize_more_blocks
[translation:info]     self.specialize_block(block)
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 301, in specialize_block
[translation:info]     hop.setup()  # this is called from here to catch TyperErrors...
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 629, in setup
[translation:info]     rtyper.call_all_setups()  # compute ForwardReferences now
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 247, in call_all_setups
[translation:info]     r.setup_final()
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rmodel.py", line 75, in setup_final
[translation:info]     self._setup_repr_final()
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rclass.py", line 586, in _setup_repr_final
[translation:info]     self.check_graph_of_del_does_not_call_too_much(graph)
[translation:info]    File "/path/to/pixie/externals/pypy/rpython/rtyper/rclass.py", line 880, in check_graph_of_del_does_not_call_too_much
[translation:info]     (graph, '\n\t'.join(lst[::-1])))
[translation:ERROR] TyperError: the RPython-level __del__() method in <FunctionGraph of (pixie.vm.custom_types:89)CustomTypeInstance.__del__ at 0x7282288> calls:
[translation:ERROR] 	(pixie.vm.custom_types:89)CustomTypeInstance.__del__
[translation:ERROR] 	(pixie.vm.object:10)FinalizerRegistry.register
[translation:ERROR] 	(rpython.flowspace.specialcase:81)rpython_print_newline
[translation:ERROR] 	(?:2)write
[translation:ERROR] 	(rpython.rlib.rposix:438)write
[translation:ERROR] 	(rpython.rtyper.lltypesystem.rffi:238)write__Signed_arrayPtr_Signed_star_3
[translation:ERROR] 	(rpython.rtyper.lltypesystem.rffi:2)ccall_write__INT_arrayPtr_Unsigned
[translation:ERROR] 	(pixie.vm.threads:107)after_external_call
[translation:ERROR] .. (pixie.vm.code:869)W_new__._lambda_
[translation:ERROR] .. block@6 with 1 exits
[translation:ERROR] .. v254 = simple_call((function _new__args), args_0)
[translation] start debugger...
> /path/to/pixie/externals/pypy/rpython/rtyper/rclass.py(880)check_graph_of_del_does_not_call_too_much()
-> (graph, '\n\t'.join(lst[::-1])))
(Pdb+) 
```

Hi @thomasmulvaney, thanks for the suggestion. It's very likely that the build I pulled was bad as this time, a different error message appears.

```
[platform:execute] gcc /tmp/usession-default-3/platcheck_20.o -pthread -Wl,--export-dynamic -lffi -lrt -o /tmp/usession-default-3/platcheck_20
Traceback (most recent call last):
  File "externals/pypy/rpython/bin/rpython", line 20, in <module>
    main()
  File "/path/to/pixie/externals/pypy/rpython/translator/goal/translate.py", line 217, in main
    targetspec_dic, translateconfig, config, args = parse_options_and_load_target()
  File "/path/to/pixie/externals/pypy/rpython/translator/goal/translate.py", line 156, in parse_options_and_load_target
    targetspec_dic = load_target(targetspec)
  File "/path/to/pixie/externals/pypy/rpython/translator/goal/translate.py", line 98, in load_target
    mod = __import__(specname)
  File "target.py", line 311, in <module>
    rt.init()
  File "pixie/vm/rt.py", line 72, in init
    import pixie.vm.threads
  File "pixie/vm/threads.py", line 9, in <module>
    from rpython.rlib.objectmodel import invoke_around_extcall
ImportError: cannot import name invoke_around_extcall
Makefile:22: recipe for target 'build_with_jit' failed
```

Well, the version before that, 81257, doesn't build either:

```
[translation:info] RTyping...
[ac1] {translation-task
starting rtype_lltype
***[ad1] translation-task}

[Timer] Timings:
[Timer] annotate                       --- 25.9 s
[Timer] rtype_lltype                   ---  0.2 s
[Timer] =========================================
[Timer] Total:                         --- 26.1 s
[translation:info] Error:
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/translator/goal/translate.py", line 318, in main
[translation:info]     drv.proceed(goals)
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/translator/driver.py", line 550, in proceed
[translation:info]     result = self._execute(goals, task_skip = self._maybe_skip())
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/translator/tool/taskengine.py", line 114, in _execute
[translation:info]     res = self._do(goal, taskcallable, *args, **kwds)
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/translator/driver.py", line 279, in _do
[translation:info]     res = func()
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/translator/driver.py", line 346, in task_rtype_lltype
[translation:info]     rtyper.specialize(dont_simplify_again=True)
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 174, in specialize
[translation:info]     self.specialize_more_blocks()
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 214, in specialize_more_blocks
[translation:info]     self.specialize_block(block)
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 301, in specialize_block
[translation:info]     hop.setup()  # this is called from here to catch TyperErrors...
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 629, in setup
[translation:info]     rtyper.call_all_setups()  # compute ForwardReferences now
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rtyper.py", line 247, in call_all_setups
[translation:info]     r.setup_final()
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rmodel.py", line 75, in setup_final
[translation:info]     self._setup_repr_final()
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rclass.py", line 586, in _setup_repr_final
[translation:info]     self.check_graph_of_del_does_not_call_too_much(graph)
[translation:info]    File "/Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rclass.py", line 880, in check_graph_of_del_does_not_call_too_much
[translation:info]     (graph, '\n\t'.join(lst[::-1])))
[translation:ERROR] TyperError: the RPython-level __del__() method in <FunctionGraph of (pixie.vm.custom_types:89)CustomTypeInstance.__del__ at 0x10b977ad0> calls:
[translation:ERROR] 	(pixie.vm.custom_types:89)CustomTypeInstance.__del__
[translation:ERROR] 	(pixie.vm.object:10)FinalizerRegistry.register
[translation:ERROR] 	(rpython.flowspace.specialcase:81)rpython_print_newline
[translation:ERROR] 	(?:2)write
[translation:ERROR] 	(rpython.rlib.rposix:438)write
[translation:ERROR] 	(rpython.rtyper.lltypesystem.rffi:238)write__Signed_arrayPtr_Signed_star_3
[translation:ERROR] 	(rpython.rtyper.lltypesystem.rffi:2)ccall_write__INT_arrayPtr_Unsigned
[translation:ERROR] 	(pixie.vm.threads:107)after_external_call
[translation:ERROR] .. (pixie.vm.custom_types:3007)CustomType30.get_field_by_idx
[translation:ERROR] .. block@459 with 2 exits(v32)
[translation:ERROR] .. v33 = eq(idx_0, (23))
[translation] start debugger...
> /Users/pbergero/source/pixie/externals/pypy/rpython/rtyper/rclass.py(880)check_graph_of_del_does_not_call_too_much()
-> (graph, '\n\t'.join(lst[::-1])))
(Pdb+)
make: *** [build_with_jit] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ lib:
 $(EXTERNALS)/pypy:
 	mkdir $(EXTERNALS); \
 	cd $(EXTERNALS); \
-	curl https://bitbucket.org/pypy/pypy/get/default.tar.bz2 >  pypy.tar.bz2; \
+	curl https://bitbucket.org/pypy/pypy/get/81254.tar.bz2 >  pypy.tar.bz2; \
 	mkdir pypy; \
 	cd pypy; \
 	tar -jxf ../pypy.tar.bz2 --strip-components=1

